src/content/en/tools/chrome-devtools/iterate/inspect-styles/css-animations.markdown

```diff
@@ -0,0 +1,177 @@
+---
+
+<p class="intro">Inspect and modify CSS animations with the Chrome DevTools
+Animation Inspector.</p>
+
+{% include shared/toc.liquid %}
+
+{% include shared/takeaway.liquid list=page.key-takeaways %}
+
+## Overview {#overview}
+
+The Animation Inspector is grouped into four main sections (or panes). This
+guide refers to each pane as follows:
+
+1. **Controls**. From here you can clear all currently recorded Animation 
+   Groups, or change the speed of the currently selected Animation Group. 
+2. **Overview**. Select an Animation Group here to inspect 
+   and modify it in the **Details** pane.
+3. **Timeline**. Pause and start an animation from here, or jump to a specific 
+   point in the animation.
+4. **Details**. Inspect and modify the currently selected
+   Animation Group. 
+
+![annotation Animation Inspector](imgs/annotated-animation-inspector.png)
+
+## Open the Animation Inspector {#open}
+
+There are two ways to open the Animation Inspector:
+
+* Go to the **Styles** pane (on the **Elements** panel) and press the 
+  **Animations** button (![animations 
+  button](imgs/animations-button.png){:.inline}). 
+* Open the Command Menu and type `Drawer: Show Animations`. 
+
+The Animation Inspector opens up as a tab next to the Console Drawer. Since
+it's a Drawer tab, you can use it from any DevTools panel. 
+
+![Empty Animation Inspector](imgs/empty-ai.png)
+
+## Record an animation {#record}
+
+The Animation Inspector automatically records all animations that it detects. 
+If the Animation Inspector detects that multiple animations are related, it
+sorts them into an **Animation Group**. 
+
+{% animation /web/tools/chrome-devtools/iterate/inspect-styles/animations/record-animations.mp4 %}
+
+If you open the Animation Inspector while an animation is already in progress,
+the Animation Inspector may not detect the animation. Reload the page while
+DevTools and the Animation Inspector are open, and the animation should get
+detected and recorded.
+
+### How are Animation Groups determined?
+
+Animations are grouped by start time (excluding delays, and so on). So, if a 
+group of animations are all triggered in the same script block, they'll get 
+registered as a group. But if they're asynchronous, they'll end up in 
+different groups. 
+
+## Inspect and modify animations {#inspect}
+
+From the **Overview** pane, click on an Animation Group to view its details. 
+The currently selected group is highlighted blue in the **Overview** pane.
+In the **Details** pane each element that has an animation applied to it 
+gets its own row.
+
+![animation group details](imgs/animation-group-details.png)
+
+The **Details** pane provides quite a bit of information about each element:
+
+1. The first part of the label, color-coded purple, is the type of element
+   which the animation is applied to.
+1. The second part of the label, color-coded gold, is the CSS 
+   selector in which the animation is applied. 
```

Comment:
Don't think this is important. Just a consistent element reference that we use throughout DevTools. Wouldn't say anything more than that's the element, click/right click to go to select/reveal that element.